src/components/ML/About/index.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from 'next/image';
+import BootCampImg from '@/public/ML/bootcamp.png';
+
+export default function About_ML() {
+  return (
+    <section className="flex flex-col items-center justify-center space-y-10 text-white">
+      <div className="flex flex-col gap-6 xl:gap-12">
+        <h1 className="text-center text-white text-4xl font-medium tracking-widest md:text-5xl lg:text-7xl">
```

Comment:
Small change for consistency from the home page title colouring. Use instead text-[#d9d9d9].
Similarly to the other sections of this page.